package.json

```diff
@@ -33,7 +33,7 @@
     "homebridge": "bin/homebridge"
   },
   "engines": {
-    "node": ">=10.17.0"
+    "node": "^16.20.2 || ^18.17.1"
```

Comment:
The synology package is using a slightly older Node 18 package nodejs.org 18.15.0-1008

So minimum version needs to take this into account

Reply:
Hi @NorthernMan54 I set minimum engine node version to 18.15.0.

I would preferably like to push this into a beta branch of homebridge rather than master, but not totally sure how it works on this repo